.github/workflows/ci.yaml

```diff
@@ -33,8 +47,6 @@ jobs:
           username: sqrereadonly
           password: ${{ secrets.DOCKERHUB_SQREREADONLY_TOKEN }}
 
-
       - name: Check tex builds
```

Comment:
Hmm, we may want to do Wil's tex checking script in one matrix rather than all three because it can be a bit slow downloading that docker image.

Reply:
Good idea, done.